README.md

```diff
@@ -26,10 +26,10 @@ Blade is the Design System that powers [Razorpay](https://razorpay.com/).
 
 ## ✨ Features
 - Cross-Platform (Works Natively on [React Web](https://blade.razorpay.com/?path=/docs/guides-installation--page#%EF%B8%8F-installation) and [React Native](https://blade.razorpay.com/?path=/docs/guides-installation--page#react-native-projects))
```

Comment:
can you fix these links as well? seems like they are broken too

Reply:
done.